code/game/objects/items/tools/crowbar.dm

```diff
@@ -126,6 +135,10 @@
 	tool_behaviour = (active ? TOOL_WIRECUTTER : TOOL_CROWBAR)
 	balloon_alert(user, "attached [active ? "cutting" : "prying"]")
 	playsound(user ? user : src, 'sound/items/change_jaws.ogg', 50, TRUE)
+	if(tool_behaviour == TOOL_CROWBAR)
+		RemoveElement(/datum/element/cuffsnapping, snap_time_weak_handcuffs, snap_time_strong_handcuffs)
```

Comment:
why the extra args?

Reply:
`RemoveElements` needs arguments to determine which bespoke element instance to remove